src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -802,17 +802,27 @@ def get_num_units(self) -> int:
         return self.sorting.get_num_units()
 
     ## extensions zone
-
-    def compute(self, input, save=True, **kwargs):
+    def compute(self, input, save=True, extension_params=None, **kwargs):
         """
         Compute one extension or several extensiosn.
         Internally calls compute_one_extension() or compute_several_extensions() depending on the input type.
 
         Parameters
         ----------
         input: str or dict or list
-            If the input is a string then computes one extension with compute_one_extension(extension_name=input, ...)
-            If the input is a dict then compute several extensions with compute_several_extensions(extensions=input)
+            The extesions to compute, which can be passed as:
+
+            * a string: compute one extension. Additional parameters can be passed as key word arguments.
+            * a dict: compute several extensions. The keys are the extension names and the values are the extension parameters.
+            * a list: compute several extensions. The list contains the extension names. Additional parameters can be passed with the extension_params
+              argument.
+        save: bool, default: True
+            If True the extension is saved to disk
```

Comment:
Also this is confusing. Because save=True only works if format != 'memory'. I have an issue open on this currently #2507. It doesn't have to be in this PR, but I think we need to find a way to be more explicit about how `save` works so that people don't think that in memory mode they can just set `save=True` and things will save. Again I think this needs to probably be in the docstring as well as in our docs.